skyrim-platform/src/platform_se/codegen/convert-files/Definitions.txt

```diff
@@ -494,6 +494,175 @@ export const enum DxScanCode {
     MouseWheelDown
 }
 
+export const enum FormType {
```

Comment:
Please add a link to creationkit.com as a comment

Reply:
I usually add comments as TSDoc.

Is that acceptable?

For example:
```ts
/** * Values were taken from {@link https://www.creationkit.com/index.php?title=GetType_-_Form} */
```

Reply:
Yes